pom.xml

```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>24.1.1</version>
+            <version>20.0</version>
```

Comment:
Why you need to downgrade the version?

Reply:
agree, no need to degrade it, just noticed it :). I will change it.

Reply:
Hi, the version should be 24.1.1-jre instead of 24.1.1

Reply:
done!